src/utils/apis/InterviewAPI.js

```diff
@@ -17,11 +17,13 @@ export const interviewAPI = {
     },
 
     getNotesByInterviewId: (interviewId, password) => {
-        return checkedFetch(`${recruitmentServiceBasicAPILink}/api/applications/get_notes?interview_note=${interviewId}`, {
+        const fetchToUse = password ? checkedFetch : authFetch
+        const headers = password ? {
+            'x-dev-password': password
```

Comment:
Nie przydałby się tu content/type json, czy jak to się tam zwało XD

Reply:
To jest GET, więc nie jest potrzebny. Ogólnie content Type mówi serwerowi, co Ty wysyłasz jako front. Jeśli jest to json to powinieneś powiedzieć, bo wtedy serwer jest w stanie przekierować Cię do odpowiedniego endpointa. Z racji, że my nic tu nie wysyłamy, tylko pytamy o dane, to na luzie nie trzeba content Type dodawać.

BTW to "Content-Type": "application/json"

Reply:
Kurde wydawało mi się że to mówi w jakim formacie ma być odpowiedź serwera. Całe życie w błędzie.